src/main/flight/mixer.c

```diff
@@ -64,6 +64,7 @@ PG_REGISTER_WITH_RESET_TEMPLATE(mixerConfig_t, mixerConfig, PG_MIXER_CONFIG, 0);
 PG_RESET_TEMPLATE(mixerConfig_t, mixerConfig,
     .mixerMode = TARGET_DEFAULT_MIXER,
     .yaw_motors_reversed = false,
+    .max_neg_thrust_pwm = 1000,
```

Comment:
There seems to be a large chunk of changes missing - `max_neg_thrust_pwm` is not defined.